internals/core_models.py

```diff
@@ -253,6 +253,7 @@ class Stage(ndb.Model):
   ot_stage_id = ndb.IntegerProperty()
 
   #Enterprise
+  rollout_impact = ndb.IntegerProperty(default=2) # default to "Medium impact"
```

Comment:
I assume you don't need to backfill any existing enterprise features because there are none on prod.